File: space triangle/code9.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int a = 0 ; a < i ; a++){

			printf("\t");
		}
	
		for(int j = 0; j < row - i ; j++){

			
			printf("%c\t",  64 + row  - j);
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/10/code9.java
import java.util.Scanner;

class TriangleDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		int temp =1;


		for(int i = 0; i < row ; i++){

			for(int j = 0 ; j < row - i ; j++ ){

				System.out.print((char)(row-j+64)+"		");
				
			}

				System.out.print("\n\n");

		}

	}

}
